packages/commerce-sdk-react/src/hooks/useAuthHelper.ts

```diff
@@ -40,13 +51,32 @@ export function useAuthHelper<Mutation extends AuthHelper>(
 > {
     const auth = useAuthContext()
     if (!auth[mutation]) throw new Error(`Unknown login helper mutation: ${mutation}`)
```

Comment:
```suggestion
    if (!auth[mutation]) throw new Error(`Unknown auth helper: ${mutation}`)
```
Just noticed we missed a spot when renaming the helpers.